scripts/create_backup_pd

```diff
@@ -21,7 +21,7 @@ EOF
 }
 
 function cleanup {
-  if [[ ${LOOP_DEVICE} == /dev/loop* ]]; then 
+  if [[ -n ${LOOP_DEVICE} ]]; then 
     if [[ $(sudo mount | grep ${LOOP_DEVICE}) == ${LOOP_DEVICE}* ]]; then 
```

Comment:
you don't need sudo here and can t you just do:
mount | grep ${LOOP_DEVICE}
[[ $? -eq 0 ]] &&  sudo umount ${LOOP_DEVICE}